tests/roles/backend_services/defaults/main.yaml

```diff
@@ -18,3 +18,15 @@ heat_stack_domain_admin_password: ''
 swift_password: ''
 dns_lb_ip: 192.168.122.80
 dns_server_ip: 192.168.122.1
+
+# ospdo env:
+# Whether source env is OSPD Director Operator
+# i.e. https://github.com/openstack-k8s-operators/osp-director-operator
+ospdo_src: false
+# rhoso namespace
+rhoso_namespace: "openstack"
+# director operator namespace
+org_namespace: "ospdo_openstack"
+# adoption repo default location
+dpa_dir: "/home/zuul/src/github.com/openstack-k8s-operators/data-plane-adoption"
+dpa_tests_dir: "{{ dpa_dir }}/tests"
```

Comment:
The `dpa_dir` and `dpa_tests_dir` variables weren't in vars.sample.yaml or any defaults.yaml in #444 so i missed that we're creating them. Let's get rid of these two and keep using the same relative paths approach that we're using?

At least based on my understanding, there shouldn't be a reason to create these variables, and while they may be passing the checks right now, having them defined this way will break running the test suite locally. Going back to relative paths would fix it.

Reply:
The reason for those was that while the original test suite scripts were running on the controller, the ospdo version is executed from controller node but running on the ospdo dedicated BM machine - the adoption_host.
so that relative scripts paths won't have the same meaning. 
```
tests/playbooks/test_minimal.yaml:
  hosts: "{{ adoption_host | default('localhost') }}"

```
https://github.com/openstack-k8s-operators/data-plane-adoption/pull/688/files#diff-59a73c8a700093e77d026bfd1c67a98f868a6303fbc922472160294b35918fd4R5

Reply:
As an alternative I can move the scripts requiring that variable into the specific ospdo files and make them execute on the condition of ospdo_src=true so htat the original code would remain the same and I'll have an ospdo version of those tasks ?

Reply:
Ok i see what you mean, we can perhaps add the variables then, but can we try using relative defaults so that the default behavior does not depend on providing absolute paths? I think one of the goals we should have in mind is keeping the configuration of the test suite as simple as possible -- not asking the user for configuring variables that can have sane defaults that "just work" in most test envs.

```
dpa_dir: "../.."
dpa_tests_dir: "{{ dpa_dir }}/tests"
```

Reply:
sure , sounds good, I'll make the adjustments